corehq/ex-submodules/casexml/apps/phone/data_providers/case/livequery.py

```diff
@@ -282,11 +283,13 @@ def take(n, iterable):
         # https://docs.python.org/2/library/itertools.html#recipes
         return list(islice(iterable, n))
 
+    add_load = case_load_counter("livequery_restore", accessor.domain)
```

Comment:
Nit: Even though this is pretty local, I'm wondering if to a random person coming across this later, `add_load` sounds too much like it's somehow acting upon the system rather than sending a measurement. Maybe something like `send_load_to_datadog`? Up to you

Reply:
388cc1e04aeca113f2070c92b2ba1b50c39d4a07